### PR TITLE
Revise previous commit and add pictures

### PR DESCRIPTION
**Pull request by bubaeee**
made revisions and corrections to previous commit

### CODE REVIEW

**Review by atreil — APPROVED** [review]
Awesome job! Thanks for addressing my feedback :)

**Comment by bubaeee** [comment]
sorry, I forgot to merge before pushing my week 3 work, so it's part of this pull request.

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -44,3 +44,115 @@ function getReturning() {
   });
 }
 
+function createMap() {
```
Comment (atreil) [inline portfolio/src/main/webapp/script.js:47]: This function is not being used anywhere. I think you can just get rid of it.

**Review by atreil — APPROVED** [review]
You almost got it! You still need to add a size to the map because otherwise it does not get rendered. Check out the style section in https://developers.google.com/maps/documentation/javascript/adding-a-google-map#understanding_the_code.

**Pull request merged**